app/src/main/java/org/gdg/frisbee/android/event/EventActivity.java

```diff
@@ -110,13 +98,12 @@ public int getItemPosition(Object object) {
 
         @Override
         public int getCount() {
-            int count = mContext.getResources().getStringArray(R.array.event_tabs).length;
```

Comment:
It's midnight but I was sick the last few days and in need of some code reviews (and fun)!